pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -1248,20 +1355,12 @@ def TS_low_slurm_prepare(self, slurmfilepath):
             print(f'--- WARNING: The memory-per-cpu on the low-res boxes SLURM script is configured for 6 seeds, not {self.nSeeds}.')
 
 
-    def TS_low_slurm_submit(self, qos='normal', A=None, t=None):
+    def TS_low_slurm_submit(self):
         # ---------------------------------
         # ----- Run turbSim Low boxes -----
         # ---------------------------------
         # Submit the script to SLURM
-        options = f"--qos='{qos}' "
-        if A is not None:
-            options += f'-A {A} '
-        if t is not None:
-            options += f'-t {t} '
-
-        sub_command = f"sbatch {options}{self.slurmfilename_low}"
-        print(f'Calling: {sub_command}')
-        _ = subprocess.call(sub_command, cwd=self.path, shell=True)
+        _ = subprocess.call(f'sbatch {self.slurmfilename_low}', cwd=self.path, shell=True)
 
```

Comment:
Unclear why you would remove extra options.

Reply:
Sorry I was trying to run this on a linux machine and was playing with the options